.github/workflows/pr.yml

```diff
@@ -1,41 +1,31 @@
 name: Pull Request Checks
 
 on:
-    pull_request:
-      branches:
-        - main
+  pull_request:
+    branches:
+      - main
 
 jobs:
-    build-and-deploy:
-      runs-on: ubuntu-20.04
-      env:
-          CI: true
-          NODE_ENV: production
-      steps:
-        - uses: actions/checkout@v3
-        - name: Set up Node.js
-          uses: actions/setup-node@v3
-          with:
-            node-version: '18.x'
-            cache: 'yarn'
-        - name: Docusaurus Webpack cache
-          uses: actions/cache@v3
-          with:
-            path: node_modules/.cache
-            key: ${{ runner.os }}-webpack-cache
-        - name: Install dependencies
-          run: yarn install --frozen-lockfile
-        - name: Build the Docusaurus site
-          env:
-            NODE_OPTIONS: "--max-old-space-size=8192"
-          run: yarn build
-    spellcheck:
-      runs-on: ubuntu-latest
-      steps:
+  build-and-deploy:
+    uses: SumoLogic/sumologic-documentation/.github/workflows/build_and_deploy.yml@main
```

Comment:
I don't think we can use this workflow this way. It requires `S3_BUCKET_NAME`, `CLOUDFRONT_DISTRIBUTION_ID`, `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`

Reply:
Not sure if we need `build-and-deploy` at all if we have `deploy-to-staging` below

Reply:
thank you!